java/code/src/com/redhat/rhn/manager/action/ActionManager.java

```diff
@@ -925,16 +936,31 @@ private static DataResult getActions(User user, PageControl pc, String mode,
         if (pc != null) {
             return makeDataResult(params, params, pc, m);
         }
-        int limit = ConfigDefaults.get().getActionsDisplayLimit();
-        if (limit > 0) {
-            m.setMaxRows(limit);
-        }
+	if (!noLimit) {
+            int limit = ConfigDefaults.get().getActionsDisplayLimit();
+            if (limit > 0) {
+                m.setMaxRows(limit);
+            }
+	}
         DataResult dr = m.execute(params);
         dr.setTotalSize(dr.size());
         dr.setElaborationParams(params);
         return dr;
     }
 
+    /**
+     * Helper method that does the work of getting a specific
+     * DataResult for scheduled actions.
+     * @param user The user in question
+     * @param pc The details of which results to return
+     * @param mode The mode
+     * @return Returns a list containing the actions for the user
+     */
+    private static DataResult getActions(User user, PageControl pc, String mode,
+            String setLabel) {
+        return getActions(user, pc, mode, null, false);
```

Comment:
I guess we want to pass the `setLabel` too to keep the older signature working the same

Reply:
Ooops, I missed that. Thanks! :+1: